src/Directory.Build.targets

```diff
@@ -1,15 +1,14 @@
 <Project>
 
   <!--
-    HACK
+      HACK:
+      There is an issue in the current preview of .NET 5 SDK where the compiler
+      that was shipped with it doesn't support source generators.
 
-      There is an issue in the current .NET 5 SDK where the compiler that was shipped
-      with it doesn't support source generators. We work this around by specifically
-      depending on a later version of the compiler.
+      We work this around by specifically depending on a later version of the compiler.
   -->
-
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Compilers" Version="3.6.0-4.final" />
+    <PackageReference Include="Microsoft.NET.Compilers.ToolSet" Version="3.6.0-4.final" />
```

Comment:
For now, the `Microsoft.NET.Compilers` package only contains `.NET Framework` edition of the compilers. Using the new `Microsoft.NET.Compilers.ToolSet` contains both `.NET Core` and framework editions of the compilers.

This should fix _rare_ build issues in non-Windows systems that use only .NET Core SDK. (See #137)